feconf.py

```diff
@@ -47,6 +47,7 @@
 else:
     raise Exception('Invalid platform: expected one of [\'gae\']')
 
+CLASSIFIERS_PKG = 'core.domain.classifier'
```

Comment:
Right now I'm trying to make the interaction_registry's approach of importing modules using "pkgutils.itermodules ...' work (I couldn't make it work initially and so I had changed the approach). Btw, I'm almost done and these changes will be reflected in my next commit.